php/util/class-fieldmanager-util-validation.php

```diff
@@ -222,9 +222,11 @@ public function add_validation() {
 			}
 
 			// Fields that should always be ignored.
-			$ignore[] = '.fm-autocomplete';
+			$ignore = array();
```

Comment:
This could be just `[]` to follow the PHP 7+ conventions.

Reply:
FM supports PHP <7, so we should not use PHP7 syntax